### PR TITLE
Try importing every file as nml and protobuf before failing

### LINKED ISSUE #4372: Zip import in tracing view should be either supported or have a better error message
Right now, protobuf is assumed when the ending is not NML.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR solves multiple small issues. 
#4372 improves the import to try both, nml and protobuf, import in tracing views.
#4379 fixes opening view only dataset links in arbitrary mode.
#4327 Adds a loading indication to the reload button in the dataset actions of a dataset in the dashboard

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
For #4372:
- Open a skeleton tracing.
- Try dropping a valid NML file with the extensions `.nml`, `.xml` and `.some_other_format`. Every single import should work.
- Do the same with a valid protobuf file.
- Try importing an invalid document like a `pdf` file. An error toast should pop up.

For #4379:
- Open a dataset in only view mode, switch to any arbitrary mode.
- Then take the link from the current tab and open it in another tab. The initial view mode should be the arbitrary view mode you chose before.

For #4327:
> This disables the edit and reload link/button in the dataset actions in the dashboard while one reload is active.

- please also check whether this is working.

### Issues:
- fixes #4372, #4379, #4327

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by MichaelBuessemeyer** [comment]
I just added the fix for #4379

**Comment by MichaelBuessemeyer** [comment]
I also added the code changes for #4327.
This disables the edit and reload link/button in the dataset actions in the dashboard while one reload is active.

**Comment by MichaelBuessemeyer** [comment]
@philippotto According to the title of #4327  all view links also be disabled.

Why is that so? I think clicking on a view link will open a tracing that gets the new dataset data from the server, right? Thus I just disabled the edit link, when the data of the dataset is being reloaded. Or am I wrong here and the view links should also be disabled?

**Comment by MichaelBuessemeyer** [comment]
@philippotto Could you please review this PR again? Now all actions are disabled, when the dataset is being reloaded

File: frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js [hunk 1]
```diff
@@ -51,20 +57,38 @@ class DatasetActionView extends React.PureComponent<Props, State> {
...
       verticalAlign: "middle",
     };
+    if (isReloading) {
+      // We need to explicitly grayscale the images when the dataset is being reloaded.
+      centerBackgroundImageStyle.filter = "grayscale(100%) opacity(25%)";
+    }
+    const disabledWhenReloadingStyle = isReloading
+      ? { pointerEvents: "none", color: "rgba(0, 0, 0, 0.25)" }
+      : null;
+    const disabledWhenReloadingAction = e => (isReloading ? e.preventDefault() : null);
```
[the remark this answers is not in the document]
Comment (MichaelBuessemeyer) [reply frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js]: `preventDefault` disables the whole link. Thus we only need to use it, when the dataset is being reloaded. Keeping it all the time would always disable all `Links`/Actions.

[the remark this answers is not in the document]
Comment (philippotto) [reply frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js]: Ah, I see, nevermind then!

File: frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js [hunk 2]
```diff
@@ -124,44 +124,61 @@ export async function importTracingFiles(files: Array<File>, createGroupForEachF
...
+          tryImportFunctions = [tryParsingFileAsNml, tryParsingFileAsProtobuf];
+        } else {
+          tryImportFunctions = [tryParsingFileAsProtobuf, tryParsingFileAsNml];
+        }
+        let maybeImportAction = await tryImportFunctions[0](file);
+        maybeImportAction = maybeImportAction || (await tryImportFunctions[1](file));
+        if (!maybeImportAction) {
+          throw new Error(`"${file.name}" could not be parsed as either NML or protobuf.`);
         }
+        return maybeImportAction;
```
[the remark this answers is not in the document]
Comment (MichaelBuessemeyer) [reply frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js]: I agree with you. Your version is definitely easier to read. Unfortunately, I had to disable on eslint rule for the loop :/

**Comment by MichaelBuessemeyer** [comment]
@philippotto I applied all your feedback. Could you please review the newest version?

**Comment by MichaelBuessemeyer** [comment]
@philippotto I applied all your feedback. Could you please review the newest version?

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,17 +11,30 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
...
+[Commits](https://github.com/scalableminds/webknossos/compare/20.02.0...20.03.0)
+
+### Highlights
+- Added support for datasets with more layers than the hardware can render simultaneously. The user can disable layers temporarily to control for which layers the GPU resources should be used. [#4424](https://github.com/scalableminds/webknossos/pull/4424)
+- Time tracking precision is improved. [#4445](https://github.com/scalableminds/webknossos/pull/4445)
+
+### Added
+- Added support for datasets with more layers than the hardware can render simultaneously. The user can disable layers temporarily to control for which layers the GPU resources should be used. [#4424](https://github.com/scalableminds/webknossos/pull/4424)
+- Added a notification when downloading nml including volume that informs that the fallback data is excluded in the download. [#4413](https://github.com/scalableminds/webknossos/pull/4413)
+- Added a simpler method to install webKnossos on an own server. [#4446](https://github.com/scalableminds/webknossos/pull/4446)
```
Comment (philippotto) [inline CHANGELOG.md]: This seems to be an accidental change.

**Pull request merged**